knowledge-base/grid-binding-to-expando-object.md

```diff
@@ -188,82 +185,395 @@ In addition to the sample below, there is a [complete runnable project in GitHub
 
 ## Notes
 
-* You can also use [nested Grid model properties]({%slug grid-use-navigation-properties%}) that are `ExpandoObject`s. However, data operations like filtering, sorting and grouping are not supported for those nested properties. [Disable these features per column]({%slug components/grid/columns/bound%}#data-operations) if they are enabled for the Grid.
-* If any property in the `ExpandoObject` is `nullable`, then do not set [`EditorType`]({%slug components/grid/editing/overview%}#customize-the-editor-fields) for the column (this may be supported in the future). Instead, use a [Grid column `EditorTemplate`]({%slug grid-templates-editor%}), for example:
+The following list contains information about requirements and limitations when binding the Grid to `ExpandoObject`. They are related to using nullable data, editing, filtering, grouping, and nested model properties. A [runnable example](#example) is available afterwards.
```

Comment:
Afterwards typically refers to time rather than a specific position within a document.
```suggestion
The following list contains information about requirements and limitations when binding the Grid to `ExpandoObject`. They are related to using nullable data, editing, filtering, grouping, and nested model properties. A [runnable example](#example) is available later in the article.
```